importers/adium.js

```diff
@@ -9,7 +9,7 @@ var parseString = require('xml2js').parseString;
 
 var RemoteStorage = require("remotestoragejs");
 var ChatMessages = require("remotestorage-module-chat-messages");
-var remoteStorage = new RemoteStorage({ modules: [ ChatMessages.default ] });
+var remoteStorage = new RemoteStorage({ caching: false, modules: [ ChatMessages.default ] });
```

Comment:
According to https://remotestoragejs.readthedocs.io/en/latest/js-api/remotestorage.html it's `cache: false`.